### PR TITLE
Fix: Check ratings a bit more strict

### PR DESCRIPTION
**Pull request by BackEndTea**
This PR

* [x] checks if the ratings given are numeric

Casting a string like 'blabla' to an int will make it 0 in php. Which isn't really what we want here.

### CODE REVIEW

File: classes/Http/Controller/Admin/TalksController.php [hunk 1]
```diff
@@ -74,6 +74,10 @@ public function viewAction(Request $req)
 
     public function rateAction(Request $req)
     {
+        if (!is_numeric($req->get('rating'))) {
```
Comment (localheinz) [inline classes/Http/Controller/Admin/TalksController.php]: @BackEndTea 

I haven't looked into how filtering and validation is currently managed within the project, but if there's none, what do you think about using appropriate components sooner or later?

Comment (BackEndTea) [reply classes/Http/Controller/Admin/TalksController.php]: The ```BaseController``` has a ```validate``` function, Which is used exactly once, in the ```processAction``` of the ```SignupController```.

Comment (localheinz) [reply classes/Http/Controller/Admin/TalksController.php]: I was thinking more along the lines of creating specific validators for each use case.

Comment (BackEndTea) [reply classes/Http/Controller/Admin/TalksController.php]: As in, some kind of abstract validation class/interface, and in this case a ```RatingValidation``` which checks if the request object fills the pre-defined criteria?

Comment (localheinz) [reply classes/Http/Controller/Admin/TalksController.php]: Well, not exactly the entire request object, but the relevant request data?

Comment (BackEndTea) [reply classes/Http/Controller/Admin/TalksController.php]: I wouldn't mind that, but its outside of the scope for this PR

**Comment by chartjes** [comment]
Thanks @BackEndTea and @localheinz

**Pull request merged**